### PR TITLE
Resolve #430 to add a country filter and save set filters into cookies

### LINKED ISSUE #430: Save location filters of the user in session
Bildschirmfoto 2022-03-17 um 14 00 07

- [ ] Create (logical or) filters for countries and areas 
- [ ] save this filter into a cookie so that the user don't have to set the filters again if he revisites the scheduler

### PR DESCRIPTION
**Pull request by Doca**
Resolve #430

### CODE REVIEW

**Review by pitpalme — APPROVED** [review]
Good job, so far.
It filters, it remembers.

Alas - no sunshine without some slight rain.
None of this is a merge-blocker for now, because having it ready to be manually tested already is valuable. But as a reminder I'd like to note it nonetheless.

- let's not forget to remove "console.log()" calls, my DevTools console almost went crazy
- let's think about, whether it makes sense, to filter "Areas" according to selected countries
- there's l10n fixes necessary, but I havent's checked yet, wether it's untranslated strings or a more general issue: /helpdesk/ is shown localised, but filter labels are "English only".

File: scheduler/templates/helpdesk.html [hunk 1]
```diff
@@ -15,18 +15,52 @@
...
                     $scope.facilities = {{ facility_json | safe }};
                     $scope.areas = {{ areas_json | safe }};
-                    $scope.selectedAreas = [];
-                    $scope.toggleArea = function (area) {
-                        if ($scope.selectedAreas.indexOf(area) > -1) {
-                            $scope.selectedAreas.splice($scope.selectedAreas.indexOf(area),1);
+                    $scope.countries = {{ country_json | safe }};
+                    $scope.selectedCountryAndArea = [];
+                    $scope.isInCookie = function(item){
+                        console.log(item)
```
Comment (christophmeissner) [inline scheduler/templates/helpdesk.html:38]: Remove debug logging

**Review by christophmeissner — CHANGES_REQUESTED** [review]

**Comment by christophmeissner** [comment]
I wonder why this was not based onto `develop` but `django4` instead. It looks like as if it will work with Django 2 as well.

File: scheduler/templates/helpdesk.html (hunk 1 above)
Comment (Doca) [reply scheduler/templates/helpdesk.html:38]: thx

Comment (christophmeissner) [reply scheduler/templates/helpdesk.html:38]: ?

**Pull request merged**